client/src/pages/admin/tasks.tsx

```diff
@@ -415,11 +415,13 @@ function TaskModal({
         <Col span={24}>
           <Form.Item name="usedInCourses" label="Used in Courses">
             <Card>
-              {modalData?.courses?.map(({ name, isActive }) => (
-                <Tag key={name} color={isActive ? 'blue' : ''}>
-                  {name}
-                </Tag>
-              ))}
+              <div style={{ display: 'flex', flexWrap: 'wrap', rowGap: 8 }}>
```

Comment:
please use antd component https://ant.design/components/space

Reply:
> please use antd component https://ant.design/components/space

Oh, thank you, I was looking for some antd component, but missed this one

Reply:
fixed